assets/src/blocks/Spreadsheet/Spreadsheet.js

```diff
@@ -68,42 +68,60 @@ export class Spreadsheet extends Component {
               clearable={ false }
               options= { colors }
             />
+            <TextControl
+              label={__('Spreadsheet URL', 'planet4-blocks-backend')}
+              placeholder={__('Enter Google Spreadsheet URL', 'planet4-blocks-backend')}
+              value={ attributes.url }
+              onChange={ toAttribute( 'url' ) }
+            />
+            <div className="sidebar-blocks-help">
+              <ul>
+                <li>
+                  { __(`From Your Google Spreadsheet Table choose File -> Publish on web.
```

Comment:
The description has been updated for https://jira.greenpeace.org/browse/PLANET-5022, so I guess it should be updated here too?

Reply:
Good catch!